src/docs/guides/21-how-to-setup-and-use-stripe/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: How to set up and use Stripe
+slug: how-to-set-up-and-use-stripe
+updated: 2019-02-04
+category: guides
+ingress:
+  To enable payments and receive commissions in your marketplace, you
+  need a free Stripe account. This guide will help you in creating a
+  Stripe account and adding Stripe API keys to Flex Console and Flex
+  Template for Web.
+published: true
+---
+
+## 1. Create and confirm your free Stripe account
+
+[Register to Stripe](https://dashboard.stripe.com/register). After
+filling the form you will be asked the question "How do you want to get
```

Comment:
I think that every time there's quotes ("How do you want to get started with Stripe?") it should be in italics.
_"How do you want to get started with Stripe?"_